env/dev.js

```diff
@@ -1,2 +1,9 @@
 export const DEV = true;
-export const PROD = false;
\ No newline at end of file
+export const PROD = false;
+let ENV = "dev";
```

Comment:
ENV is not exported

And having said that, I wonder if we should have a `getEnv` after all, because now you can just mutate ENV directly, e.g.:

```js
import { ENV } from 'env';

ENV = 'foo';
setEnv('bar';);
```

So maybe it should be:

```js
let ENV = "dev";
export function setEnv(env) {
  ENV = env;
}
export function getEnv() {
  return ENV;
}
```

What do you think?

Reply:
missing exports - oups 🙈 fixed 👍

this will throw an error...  `Uncaught TypeError: Assignment to constant variable.`
as you can not directly modify imported values
```js
import { ENV } from 'env';

ENV = 'foo';
```